.github/workflows/build_test.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.9", "3.10", "3.11.0-alpha - 3.11.0"]
```

Comment:
Maybe use `"3.11-dev"` instead of `"3.11.0-alpha - 3.11.0"` (see [docs](https://github.com/actions/setup-python#available-versions-of-python)).

Reply:
The first beta is out. We should probably start providing nightly wheels once Cython 0.29.29 is out. I hope it is soon.

Reply:
> The first beta is out. We should probably start providing nightly wheels once Cython 0.29.29 is out. I hope it is soon.

Yes :crossed_fingers:

Reply:
Updated.